src/test/resources/Login.feature

```diff
@@ -4,9 +4,20 @@ Feature: Login
   I want to sign in
   So I can use the app
 
-  @automated
+  Scenario Outline: User can not login with invalid value
+    Given login page is opened
+    When user inputs user name as "<username>" and password as "<password>"
+    And user clicks on login button
+    Then user sees error message as "<message>" is displayed

```

Comment:
vì message lỗi giống nhau, nên c có thể truyền luôn vào step: user sees error message as "Login failed! Please ensure the username and password are valid." is displayed